server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -3781,7 +3781,7 @@ public Vlan createVlanAndPublicIpRange(final long zoneId, final long networkId,
             ipv4 = true;
         }
 
-        if (startIPv6 != null) {
+        if (vlanIp6Cidr != null) {
```

Comment:
CloudStack uses SLAAC for managing IPv6 ranges, therefore it is not necessary to have an start/end IPv6 address; on the other hand, IPv6 CIDR is mandatory for IPv6 networks.

That is why I changed this and other pieces of code that had start/end ipv6 address as mandatory.